userdocs/src/usage/iamserviceaccounts.md

```diff
@@ -97,6 +97,13 @@ The `eksctl create iamserviceaccount` command supports `--include` and `--exclud
 [this section](/usage/managing-nodegroups#include-and-exclude-rules) for more details about how these work).
 And the `eksctl delete iamserviceaccount` command supports `--only-missing` as well, so you can perform deletions the same way as nodegroups.
 
+???+ note
+    Bear in mind that IAM service accounts are scoped within a namespace, i.e. two service accounts with the same name may exist in different namespaces. Thus, to uniquely define a service account as part of `--include`, `--exclude` flags, you will need to pass the name string in the `namespace/name` format. E.g. (see config file below):
```

Comment:
>(see config file below)

Is this referring to the command example on line 104?

Reply:
yeah, it's pointing to the example config file a few lines below, I think I can remove that altogether, the example is clear enough on its own